src/components/item/item-options.ts

```diff
@@ -39,14 +40,23 @@ export class ItemOptions {
    */
   @Output() ionSwipe: EventEmitter<ItemSliding> = new EventEmitter<ItemSliding>();
 
-  constructor(private _elementRef: ElementRef, private _renderer: Renderer) {}
+  constructor(private _elementRef: ElementRef, private _renderer: Renderer, private _plt: Platform) {}
 
   /**
    * @hidden
    */
   getSides(): number {
-    if (isPresent(this.side) && this.side === 'left') {
-      return ITEM_SIDE_FLAG_LEFT;
+    if (isPresent(this.side)) {
+      switch(this.side) {
```

Comment:
Can you add whitespace to make it: `switch (this.side)` (it is giving me linter errors)